tests/e2e/tests/test/e2e.ts

```diff
@@ -19,5 +19,6 @@ export default function() {
     .then(() => _runServeAndE2e())
     .then(() => _runServeAndE2e('--prod'))
     .then(() => _runServeAndE2e('--aot'))
-    .then(() => _runServeAndE2e('--aot', '--prod'));
+    .then(() => _runServeAndE2e('--aot', '--prod'))
+    .then(() => _runServeAndE2e('--config', 'config'));
```

Comment:
Could you create an e2e that actually uses another config instead of the same default one?